submission-scripts/script-sbatch-srun/magpie.sbatch-srun-spark-with-alluxio

```diff
@@ -0,0 +1,897 @@
+#
+#  Copyright (C) 2019-2020 Intel Corporation. All rights reserved.
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License version
+#  2 as published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  Aleksander Kantak <aleksander.kantak@intel.com>
+########################################################################
+
+############################################################################
+# SLURM Customizations
+############################################################################
+
+# Node count.  Node count should include one node for the
+# head/management/master node.  For example, if you want 8 compute
+# nodes to process data, specify 9 nodes below.
+#
+# If including Zookeeper, include expected Zookeeper nodes.  For
+# example, if you want 8 Hadoop compute nodes and 3 Zookeeper nodes,
+# specify 12 nodes (1 master, 8 Hadoop, 3 Zookeeper)
+#
+# Also take into account additional nodes needed for other services.
+#
+# Many of the below can be configured on the command line.  If you are
+# more comfortable specifying these on the command line, feel free to
+# delete the customizations below.
+
+#SBATCH --nodes=<my_node_count>
+#SBATCH --output="slurm-%j.out"
+
+# Note defaults of MAGPIE_STARTUP_TIME & MAGPIE_SHUTDOWN_TIME, this
+# timelimit should be a fair amount larger than them combined.
+#SBATCH --time=<my_time_in_minutes>
+
+# Job name.  This will be used in naming directories for the job.
+#SBATCH --job-name=<my_job_name>
+
+# Partition to launch job in
+#SBATCH --partition=<my_partition>
+
+## SLURM Values
+# Generally speaking, don't touch the following, misc other configuration
+
+#SBATCH --ntasks-per-node=1
+#SBATCH --exclusive
+#SBATCH --no-kill
+
+# Need to tell Magpie how you are submitting this job
+export MAGPIE_SUBMISSION_TYPE="sbatchsrun"
+
+
+############################################################################
+# Magpie Configurations
+############################################################################
+
+# Directory your launching scripts/files are stored
+#
+# Normally an NFS mount, someplace magpie can be reached on all nodes.
+export MAGPIE_SCRIPTS_HOME="${HOME}/magpie"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp.  This will store local conf files and log files for your
+# job.  If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR option.  See README for more details.
+#
+export MAGPIE_LOCAL_DIR="/tmp/${USER}/magpie"
+
+# Magpie job type
+#
+# "spark" - Run a job according to the settings of SPARK_JOB.
+#
+# "testall" - Run a job that runs all basic sanity tests for all
```

Comment:
noticed "alluxio" wasn't listed here in the spark-with-alluxio file.  Looks like a missed addition in the Makefile.